frontend/src/metabase/containers/DataPicker/CardPicker/CardPickerContainer.tsx

```diff
@@ -38,7 +38,7 @@ interface CardPickerStateProps {
 interface CollectionsLoaderProps {
   collectionTree: Collection[];
   collections: Collection[];
-  rootCollection: Collection;
+  rootCollection?: Collection;
```

Comment:
The component handles missing `rootCollection` but the type was incorrect. Lets explicitly tell it can be `undefined` here